src/data/zwave_js.ts

```diff
@@ -83,6 +83,9 @@ export interface ZWaveJSNodeStatus {
   node_id: number;
   ready: boolean;
   status: number;
+  is_routing: boolean | null;
+  is_secure: boolean | null;
```

Comment:
Can we show `is_secure` information on the device page? (the information already seems to be present in server responses but we aren't showing it at this stage from what I see/can tell :-) , perhaps I should open a separate issue for that? )

Reply:
https://github.com/home-assistant/frontend/pull/10135